main/src/main/java/org/mobilitydata/gtfsvalidator/validator/FeedServiceDateValidator.java

```diff
@@ -38,15 +35,6 @@
   @Override
   public void validate(NoticeContainer noticeContainer) {
     for (GtfsFeedInfo feedInfo : feedInfoTable.getEntities()) {
-      if (feedInfo.hasFeedStartDate() && !feedInfo.hasFeedEndDate()) {
```

Comment:
Instead of removing the rule logic entirely, shouldn't we just downgrade the existing notice to a WARNING?

Reply:
That's a great idea, let's have a warning.

Reply:
Done in https://github.com/MobilityData/gtfs-validator/pull/735/commits/167a4cd757507539d4cdfb3ae98e686a90363625